### ISSUE TITLE
Can't start after patch.

### ISSUE THREAD

**Issue by Timsan90** [Healix/Gw2Launcher#59]

**Comment by Healix** [comment]
That'll happen if you're using -authsrv or -assetsrv with an outdated IP, so ensure you don't have those set in the arguments under the launch options (Settings > Guild Wars 2 > Launch options, and Edit > Launch options).

**Comment by Timsan90** [comment]
thank you now its working again.

**Comment by b-o-bkelso** [comment]
Hey, I have never used either of these 2 arguments and I still encounter this problem after every patch and have to create a new Local.dat for every account.

**Comment by Healix** [comment]
What exactly is the problem? Are you saying GW2 is failing to login (as if you didn't have an internet connection) after patching?

Why do you need to create a new Local.dat / what does that accomplish? Local.dat is primarily for hardware-related settings (stored login, graphics, sound, cache, file manifests). If you're simply having a connection problem, creating a new one shouldn't have any effect and would serve more as a placebo as it forces GW2 to patch.

**Comment by b-o-bkelso** [comment]
Yes, exactly what you described. After every GW2 patch I can't start any of my accounts through GW2Launcher. It's trying to update the game but claiming I wouldn't have an internet connection. The only solution I found so far is to -> Edit -> Local.dat -> Create a New File. Setting everything up again like this fixes the issue until the next GW 2patch release. Maybe it is due to the fact that my appdata folder is on a different SSD than my game folder?

**Comment by Healix** [comment]
At what point does this happen? For example, if GW2 can't connect to the patch server, it'll show a smaller gray window that will display "Connection error(s) detected. Retrying...". If it can connect, but fails to download, it'll show "Download failed! Please check your internet connection and try again". If it can connect and download, it'll move on to the launcher where you can see the login/play button. If it loses its connection at that point, or fails to download the patch, it'll get stuck trying to complete the patch.

If something else is happening, what exactly is it showing?

If you're getting stuck on "Download failed!" that's caused by patching the game outside of Gw2Launcher. Whenever there's a patch, launching an account in Gw2Launcher should cause the game to crash on startup, which Gw2Launcher should detect and automatically close, then proceed to update all accounts.

**Comment by b-o-bkelso** [comment]
It's always the second example you mentioned, the window where it tries to update and shortly after displays "Download failed! Please check your internet connection and try again". However, that window remains open and Gw2Launcher does not proceed to update the accounts, but I will try the manual Update function the next time it happens.
So to prevent this in the first place, I should always start the game through GW2Launcher?

**Comment by Healix** [comment]
If you exclusively launch through Gw2Launcher / never patch outside of Gw2Launcher, updating will be entirely automatic. 

If you patch outside of Gw2Launcher, the Local.dat files still need to be patched, so you need to right click any account > All > Update. Alternatively, you can enable checking for updates on launch under Settings > Updates.

**Comment by kncome** [comment]
So i had the same issue as  b-o-bkelso, updated outside the launcher. If you right click your account and chose "selected, launch normal", it will download the new patch without the error. You can also right click, all, launch normal. Then you'll need to close each client one by one as the patch finished to download for all you account set.
